content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/README.md

```diff
@@ -134,11 +136,17 @@ A hook attribute from the following should be added to the corresponding element
 
 The `img` and an optional image `map` should be placed inside a `noscript` element with the `data-cmp-hook-image="noscript"` attribute.
 They will be inserted into the DOM by the JavaScript component.
-
 To allow lazy loading it is expected that the `data-cmp-lazy` and `data-cmp-src` options are supplied.
+In this way a custom solution for lazy-loading is used for the browsers which don't support native lazy-loding.
+
+The browsers which support native lazy-loading will use it instead of the custom solution, with one exception. Because the custom solution is still in place, the usage of the 
+`noscript` element and the insertion of the `img` element via JS will cause the Firefox browser to preload all the images and not use the native lazy loading functionality.
+It is something relating to how the browser deals with the current implementation, because on Chrome and the on the rest of Chrome based browsers the native lazy-loading functionality works.
+For this reason, until the lazy-loading will be adopted by all major browsers by default (e.g on Safari it's an experimental feature which can be enabled), for Firefox will be used the custom lazy-loading implementation.
```

Comment:
How about using a `loading="lazy"` polyfill: https://github.com/mfranzke/loading-attribute-polyfill#loadinglazy-attribute-polyfill ?

Reply:
I used https://github.com/verlok/vanilla-lazyload, which have built in support for hybrid lazy-loading.